app/pages/sections/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -15,11 +17,21 @@ export const populateTable = ((selectedPrice) => {
   manifest.addPriceTable.data[0][1].data = selectedPrice.itemUnit.description;
 });
 
+export const formatFormData = ((populatedData) => {
+  questionExtractor('quantity', manifest).data = populatedData.quantity ? populatedData.quantity.trim() : '';
+  if (populatedData.price) {
+    manifest.addPriceTable.data[0][0].question.data = populatedData.price.trim();
+  } else {
+    manifest.addPriceTable.data[0][0].question.data = '';
+  }
+});
+
 export const getContext = ({
-  orderId, solutionName, serviceRecipientName, odsCode, selectedPrice,
+  orderId, solutionName, serviceRecipientName, odsCode, selectedPrice, populatedData = null,
```

Comment:
why is this set to a default value of null?

Reply:
becuase this is just for the post with the extra data

Reply:
mm ok but won't it be `undefined` if it's not provided? don't think defaulting it to `null` adds any value

Reply:
yeah true will get rid